lighthouse-plugin-ad-speed-insights/gatherers/async-call-stacks.js

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// @ts-ignore
+const Gatherer = require('lighthouse/lighthouse-core/gather/gatherers/gatherer.js');
+
+/**
+ * @fileoverview Tracks unused JavaScript
+ */
+class AsyncCallStacks extends Gatherer {
```

Comment:
Have you run this with core LH audits to ensure there are no unintended side effects?

Reply:
The API is virtually the same. Only difference is that the `parent` property is added.

There possibly some difference in overhead

Reply:
Everything seems fine in smoke tests.